src/disk_utils.cpp

```diff
@@ -1188,12 +1193,24 @@ int build_disk_index(const char *dataFilePath, const char *indexFilePath, const
                      "(n*(d+1)*4) bytes for storing pre-processed base vectors, "
                      "apart from the intermin indices and final index."
                   << std::endl;
-        std::string prepped_base = index_prefix_path + "_prepped_base.bin";
         data_file_to_use = prepped_base;
         float max_norm_of_base = diskann::prepare_base_for_inner_products<T>(base_file, prepped_base);
         std::string norm_file = disk_index_path + "_max_base_norm.bin";
         diskann::save_bin<float>(norm_file, &max_norm_of_base, 1, 1);
         diskann::cout << timer.elapsed_seconds_for_step("preprocessing data for inner product") << std::endl;
+        created_temp_file_for_processed_data = true;
+    }
+    else if (compareMetric == diskann::Metric::COSINE)
+    {
+        Timer timer;
+        std::cout << "Normalizing data for cosine to temporary file, please ensure there is additional "
+                     "(n*d*4) bytes for storing normalized base vectors, "
+                     "apart from the intermin indices and final index."
```

Comment:
what does "intermin indices" mean? intermediate?

Reply:
sorry i meant intermediate indices, i.e., the in-mem indices created by DiskANN before the final disk index is created.
Fixed this, thanks for pointing out!